### PR TITLE
feat(aggregation-tests): tests for aggregation API

### PR DESCRIPTION
**Pull request by williamhaley**
### Improvements

* Tests for aggregate metadata APIs

### CODE REVIEW

**Comment by williamhaley** [comment]
@craigrbarnes @ZakirG Can you review this PR? Note, it's _not_ against `master`. It's against `chore/aggregation-init`. The goal is to make it easy for you both to review tests/code coverage that I added that will allow `chore/aggregation-init` to eventually work. There should be no functional changes in this PR.

File: src/mds/agg_mds/redis_cache.py [hunk 1]
```diff
@@ -86,10 +89,13 @@ async def get_all_named_commons_metadata(self, name: str):
         return await self.json_get(name, ".metadata")
 
     async def get_commons_metadata_guid(self, name: str, guid: str):
-        idx = await self.redis_cache.execute(
```
Comment (williamhaley) [inline src/mds/agg_mds/redis_cache.py:89]: I lied, I guess there is _one_ functional change. In my testing, the `metadata[{idx}]` does not work on our flavor of JSON redis. Does that work for you @craigrbarnes? Making this a two-phase operation got things working for me while testing.

Comment (craigrbarnes) [reply src/mds/agg_mds/redis_cache.py:89]: That works for me.

**Review by craigrbarnes — APPROVED** [review]
Looks good. Thanks for adding all the tests.

**Pull request merged**